importer_utils.py

```diff
@@ -145,24 +145,31 @@ def extract_municipality_name(category_name):
     :param category_name: Category of Swedish nature reserves,
                           like "Naturreservat i Foo kommun"
     """
+    municip_cache = {}
```

Comment:
Doesn't the cache get emptied every time we call the method?

I believe you have to initialise it as a variable in util itself.

Break out as a subtask and do in a separate patch?